### ISSUE TITLE
Install on WearOS Watch Causes Loading Issue

### ISSUE THREAD

**Issue by NethCoder** [jwoglom/controlX2#16]
I was trying to install the Wear Release apk from Android Studio on a Google Pixel 1 Smartwatch (uses WearOS 4). This is the second watch wear I get this response: "Waiting on Phone".

![20231112_171538](https://github.com/jwoglom/controlX2/assets/52761543/77e1bb8f-7bf2-4b9f-9325-1ebce9bc49a2)

**Comment by NethCoder** [comment]
I should add that it _does_ work with the phone app, but it requires having the phone app do all the processing first, then displaying the results on the watch afterward. Sorry, is that the normal process?

**Comment by jwoglom** [comment]
@NethCoder yes, the watch app connects to the phone app, which is the one which actually makes the BT connection to the pump as this is implemented currently.
